scripts/O365_Kill_Active_and_Oauth_Sessions/Readme.md

```diff
@@ -0,0 +1,39 @@
+## kill_sessions.ps1
```

Comment:
This references a script which does not exist in the repo. Is this Pull Request a Work In Progress?